src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -627,6 +627,64 @@ function updateConstSumMessages() {
     }
 }
 
+function getMaxSelectableMsqChoices(qNum) {
+    const $input = $(`input[name="msqMaxSelectableChoices-${qNum}"]`);
+
+    return $input.prop('disabled') ? Number.MAX_SAFE_INTEGER : $input.val();
+}
+
+function getMinSelectableMsqChoices(qNum) {
+    const $input = $(`input[name="msqMinSelectableChoices-${qNum}"]`);
+
+    return $input.prop('disabled') ? 0 : $input.val();
+}
+
+function validateMsqQuestions() {
+    const msqQuestionNums = getQuestionTypeNumbers('MSQ');
+
+    // validate min/max selectable choices restrictions
+    for (let i = 0; i < msqQuestionNums.length; i += 1) {
+        const qNum = msqQuestionNums[i];
+        let recipientIndex = 0;
```

Comment:
I may be wrong, but the way this variable is used, this is essentially the same as `i`?

Reply:
This is not the same as `i`. They are like two loops performing an O(N^2) -

``` java
for (i = 0; i < arr.length; ++i) {
    for (j = 0; j < arr[i].length; ++j) {
        // code
    }
}
```

The role of `recipientIndex` is same as role of loop counter `j` in the above example. `i` is meant to loop over all the MSQ questions. `recipientIndex` is meant to loop over all recipients for a given MSQ question.